qiskit/pulse/reschedule.py

```diff
@@ -141,27 +141,35 @@ def add_implicit_acquires(schedule: ScheduleComponent, meas_map: List[List[int]]
     """
     new_schedule = Schedule(name=schedule.name)
 
+    # Dict[qubit, Dict[time, AcquireInstruction]]
+    acquires_map = dict()
     for time, inst in schedule.instructions:
         if isinstance(inst, AcquireInstruction):
             if any([acq.index != mem.index for acq, mem in zip(inst.acquires, inst.mem_slots)]):
                 warnings.warn("One of your acquires was mapped to a memory slot which didn't match"
                               " the qubit index. I'm relabeling them to match.")
-            cmd = Acquire(inst.duration, inst.command.discriminator, inst.command.kernel)
-            # Get the label of all qubits that are measured with the qubit(s) in this instruction
-            existing_qubits = {chan.index for chan in inst.acquires}
-            all_qubits = []
-            for sublist in meas_map:
-                if existing_qubits.intersection(set(sublist)):
-                    all_qubits.extend(sublist)
-            # Replace the old acquire instruction by a new one explicitly acquiring all qubits in
-            # the measurement group.
-            new_schedule |= AcquireInstruction(
```

Comment:
Okay, for this file, I don't think that many changes are really necessary. Why not simply replace lines 158-161 the way you replaced similar lines in the tests, by looping over `all_qubits`?